ui/base/src/main/kotlin/org/cru/godtools/base/ui/util/ModelUtils.kt

```diff
@@ -34,4 +34,5 @@ private fun Context.getToolCategoryStringRes(category: String) =
         0 -> null
         else -> resources.getString(id)
     }
+fun String.getLocalStringFromToolCategory(context: Context) = context.getToolCategoryStringRes(this)
```

Comment:
this doesn't account for backwards compatibility if a new category is added to the API. old versions of the app will then show `null` and not fallback to the new category value.